OpenRA.Game/Network/Session.cs

```diff
@@ -185,7 +185,7 @@ public class Global
 			public bool Fog = true;
 			public bool AllyBuildRadius = true;
 			public int StartingCash = 5000;
-			public string TechLevel = "none";
+			public string TechLevel = "Unrestricted";
```

Comment:
But if I don't change this, games with unrestricted tech setting show up as non-default. I didn't change the default setting, it is the same.